unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt

```diff
@@ -431,6 +431,10 @@ Show [\u20b9]
 # [\p{Alphabetic}] ∥ \p{Script=Common}
 #  & [\p{Decomposition_Type=None} \p{Decomposition_Type=Canonical}]
 
+# Nonalphabetic bindus
+Let $nonAlphaBindus = [\u0C04\u0F82\u0F83\U00011080\U00011081]
+[\p{InSc=Bindu} - $nonAlphaBindus - \p{Alphabetic}] = [] 
```

Comment:
By the way, I don't quite understand why the other formulation wasn't working, but this is a better formulation anyway, because we don't care that they are exactly equal, just that if we subtract them as well as alphabetic, we are down to zero.

It could also be expressed as:

\p{InSc=Bindu} ⊆ [$nonAlphaBindus  \p{Alphabetic}]

Reply:
Thanks, Mark! We'll revisit this after checking in with SAH as to the results.

Reply:
I think we got these five exceptions from the Unicode utilities running on Unicode 14. It's possible that the set of exceptions is slightly different now. As we discuss the TODO we could experiment with the exceptions set, such as emptying it and looking at the invariant test failure.